packages/shared/src/store/lure.ts

```diff
@@ -321,136 +220,64 @@ export function useLure({
       await fetchLure(flag, inviteServiceEndpoint, inviteServiceIsDev);
       return true;
     },
-    enabled: canCheckForUpdate && uninitialized,
-    refetchInterval: 5000,
-  });
-
-  const toggle = async () => {
-    lureLogger.crumb('toggling', flag);
-    return useLureState.getState().toggle(flag);
-  };
-
-  const describe = useCallback(() => {
-    return useLureState.getState().describe(flag);
-  }, [flag]);
-
-  lureLogger.crumb('useLure', flag, bait, lure, describe);
-
-  return {
-    ...lure,
-    supported: bait,
-    describe,
-    toggle,
-  };
-}
-
-export function useLureLinkChecked(url: string | undefined, enabled: boolean) {
-  const prevData = useRef<boolean | undefined>(false);
-  const pathEncodedUrl = stringToTa(url ?? '');
-  const { data, ...query } = useQuery({
-    queryKey: ['lure-check', url],
-    queryFn: async () =>
-      subscribeOnce<boolean>(
-        { app: 'grouper', path: `/v1/check-link/${pathEncodedUrl}` },
-        4500
-      ),
-    enabled: enabled && Boolean(url),
+    enabled: canCheckForUpdate,
     refetchInterval: 5000,
   });
 
-  prevData.current = data;
-
-  lureLogger.crumb('useLureLinkChecked', url, data);
+  lureLogger.crumb('useLure', flag, bait, lure);
 
-  return {
-    ...query,
-    good: data,
-    checked: query.isFetched && !query.isLoading,
-  };
-}
-
-export function useLureLinkStatus({
-  flag,
-  inviteServiceEndpoint,
-  inviteServiceIsDev,
-}: {
-  flag: string;
-  inviteServiceEndpoint: string;
-  inviteServiceIsDev: boolean;
-}) {
-  const [lastLoggedStatus, setLastLoggedStatus] = useState('');
-  const { supported, fetched, enabled, url, deepLinkUrl, toggle, describe } =
-    useLure({
-      flag,
-      inviteServiceEndpoint,
-      inviteServiceIsDev,
-    });
-  const { good, checked } = useLureLinkChecked(url, !!enabled);
-
-  const inviteInfo = useMemo(
-    () => ({
-      flag,
-      supported,
-      fetched,
-      enabled,
-      checked,
-      good,
-      url,
-      deepLinkUrl,
-    }),
-    [flag, supported, fetched, enabled, checked, good, url, deepLinkUrl]
-  );
-
-  lureLogger.crumb('useLureLinkStatus', inviteInfo);
+  const { fetched, url, deepLinkUrl } = lure;
 
   const status = useMemo(() => {
-    if (!supported) {
+    if (!bait) {
       return 'unsupported';
     }
 
-    if (fetched && !enabled) {
-      return 'disabled';
-    }
-
     if ((url && checkOldLureToken(url)) || (fetched && !url)) {
       return 'stale';
```

Comment:
are we still handling stale links? (previous lure version)

Reply:
let me make sure

Reply:
added something just to make sure